packages/wdio-cli/src/interface.js

```diff
@@ -208,21 +192,47 @@ export default class WDIOCLInterface extends EventEmitter {
         )
 
         this.updateClock()
+    }
 
-        if (this.sigintTriggered) {
-            clearTimeout(this.interval)
-            this.interface.log('\n')
-            this.interface.log(!isFinished
-                ? 'Ending WebDriver sessions gracefully ...\n' +
-                  '(press ctrl+c again to hard kill the runner)'
-                : 'Ended WebDriver sessions gracefully after a SIGINT signal was received!'
-            )
+    printReporters() {
+        const totalJobs = this.totalWorkerCnt
+
+        this.interface.clearAll()
+
```

Comment:
Log newline like before?

Reply:
I believe that was intentional. Pull down the repo, play with it, and tell me what you think about the formatting.

Reply:
Fixed.